ingest/pom.xml

```diff
@@ -272,11 +272,6 @@
             <artifactId>resilience4j-retry</artifactId>
             <version>${resilience4j.version}</version>
         </dependency>
-        <dependency>
```

Comment:
How do we know it's ok to remove?

Reply:
it was here only because the verify phase complained